### PR TITLE
Added Fade the background when FAB is open #2865

### LINKED ISSUE #2865: Fade the background when FAB is open
**Summary:** 
Currently, when the FAB is open it is hard to see the small camera and gallery icons, many apps like Hangouts handle this by fading the background.

**Screen-shots:** 
Screenshot_20190405-191350

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description**
Added Fade the background when FAB is open so that the tiny camera and gallery button are visible.

Fixes #2865 Fade the background when FAB is open

**Tests performed**
Tested prodDebug on Google Pixel 3 with API level 28 stock android.

**Screenshots**
screenshot-2019-04-11_23 35 03 299

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java [hunk 1]
```diff
@@ -111,12 +114,14 @@ private void animateFAB(boolean isFabOpen) {
...
         if (fabPlus.isShown()){
             if (isFabOpen) {
+                contributionsParentView.setAlpha((float) 1);
                 fabPlus.startAnimation(rotate_backward);
                 fabCamera.startAnimation(fab_close);
                 fabGallery.startAnimation(fab_close);
                 fabCamera.hide();
                 fabGallery.hide();
             } else {
+                contributionsParentView.setAlpha((float) 0.2);
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java]: Would you mind adding two (here and the setAlpha above) small comments explaining very briefly the effect in each case? For instance "Fade the gallery when FAB is open" / etc
Thanks!

Comment (madhurgupta10) [reply app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java]: @nicolas-raoul Added Comments :)

**Comment by neslihanturan** [comment]
@madhurgupta10 thanks for the PR but I don't think this improves the UI. We don't have multiple actions on background so no need to cover background in my opinion. Sorry I am closing this PR, please ayone feel free to reopen it

**Comment by madhurgupta10** [comment]
@neslihanturan Okay :)

**Pull request closed without merging**